storage-provider/server/src/pipeline/mod.rs

```diff
@@ -279,6 +284,29 @@ async fn add_piece(
     let mut sector = find_sector_for_piece(&state).await?;
 
     tracing::info!("Adding a piece...");
+
+    // NOTE(@jmg-duarte,31/01/2025): Unsure if getting the best block would be better
+    let current_block = state.xt_client.height(true).await?;
```

Comment:
Precommit depends on the last finalized block, so this should too.

Reply:
Clippy failed so I'll remove that too